<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>AddConsulta</class>
 <widget class="QMainWindow" name="AddConsulta">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1150</width>
    <height>750</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>1100</width>
    <height>700</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>1150</width>
    <height>750</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Nueva Consulta</string>
  </property>
  <property name="styleSheet">
   <string notr="true"/>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QFrame" name="frame">
    <property name="geometry">
     <rect>
      <x>39</x>
      <y>29</y>
      <width>1081</width>
      <height>801</height>
     </rect>
    </property>
    <property name="frameShape">
     <enum>QFrame::StyledPanel</enum>
    </property>
    <property name="frameShadow">
     <enum>QFrame::Raised</enum>
    </property>
    <widget class="QLabel" name="label">
     <property name="geometry">
      <rect>
       <x>440</x>
       <y>10</y>
       <width>191</width>
       <height>31</height>
      </rect>
     </property>
     <property name="text">
      <string>Nueva Consulta</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_2">
     <property name="geometry">
      <rect>
       <x>90</x>
       <y>70</y>
       <width>131</width>
       <height>21</height>
      </rect>
     </property>
     <property name="text">
      <string>ID Médico</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="inputIdMedico">
     <property name="geometry">
      <rect>
       <x>90</x>
       <y>110</y>
       <width>271</width>
       <height>31</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">background-color: rgb(255, 255, 255);
</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_3">
     <property name="geometry">
      <rect>
       <x>430</x>
       <y>70</y>
       <width>141</width>
       <height>21</height>
      </rect>
     </property>
     <property name="text">
      <string>ID Paciente</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="inputIdPaciente">
     <property name="geometry">
      <rect>
       <x>420</x>
       <y>110</y>
       <width>271</width>
       <height>31</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">background-color: rgb(255, 255, 255);
</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_4">
     <property name="geometry">
      <rect>
       <x>750</x>
       <y>70</y>
       <width>231</width>
       <height>21</height>
      </rect>
     </property>
     <property name="text">
      <string>Motivo de consulta</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_5">
     <property name="geometry">
      <rect>
       <x>90</x>
       <y>160</y>
       <width>181</width>
       <height>21</height>
      </rect>
     </property>
     <property name="text">
      <string>Fecha de visita</string>
     </property>
    </widget>
    <widget class="QLabel" name="FechaAuto">
     <property name="geometry">
      <rect>
       <x>90</x>
       <y>200</y>
       <width>181</width>
       <height>31</height>
      </rect>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
    <widget class="QPushButton" name="botonFecha">
     <property name="geometry">
      <rect>
       <x>290</x>
       <y>200</y>
       <width>201</width>
       <height>31</height>
      </rect>
     </property>
     <property name="text">
      <string>Generar Fecha</string>
     </property>
    </widget>
    <widget class="QPushButton" name="generarIdConsulta">
     <property name="geometry">
      <rect>
       <x>750</x>
       <y>170</y>
       <width>291</width>
       <height>31</height>
      </rect>
     </property>
     <property name="text">
      <string>Generar ID de consulta</string>
     </property>
    </widget>
    <widget class="QTableWidget" name="tablaConsultas">
     <property name="geometry">
      <rect>
       <x>90</x>
       <y>250</y>
       <width>911</width>
       <height>131</height>
      </rect>
     </property>
     <property name="maximumSize">
      <size>
       <width>1350</width>
       <height>1500</height>
      </size>
     </property>
     <property name="styleSheet">
      <string notr="true">*{background-color: rgb(255, 255, 255);
color: rgb(0, 0, 0);
	font: 8.5pt &quot;MS Shell Dlg 2&quot;;
}</string>
     </property>
     <row>
      <property name="text">
       <string/>
      </property>
     </row>
     <row>
      <property name="text">
       <string/>
      </property>
     </row>
     <row>
      <property name="text">
       <string/>
      </property>
     </row>
     <row>
      <property name="text">
       <string/>
      </property>
     </row>
     <row>
      <property name="text">
       <string/>
      </property>
     </row>
     <row>
      <property name="text">
       <string/>
      </property>
     </row>
     <row>
      <property name="text">
       <string/>
      </property>
     </row>
     <column>
      <property name="text">
       <string>ID Consulta</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string>Nombre médico</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string>Apellido médico</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string>Nombre Paciente</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string>Apellido Paciente</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string>Fecha de consulta</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string>Motivo de consulta</string>
      </property>
     </column>
    </widget>
    <widget class="QLabel" name="label_6">
     <property name="geometry">
      <rect>
       <x>90</x>
       <y>390</y>
       <width>301</width>
       <height>31</height>
      </rect>
     </property>
     <property name="text">
      <string>Ingrese el ID de consulta</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="inputIdConsulta">
     <property name="geometry">
      <rect>
       <x>400</x>
       <y>390</y>
       <width>301</width>
       <height>31</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">background-color: rgb(255, 255, 255);</string>
     </property>
    </widget>
    <widget class="QPushButton" name="guardarConsulta">
     <property name="geometry">
      <rect>
       <x>790</x>
       <y>390</y>
       <width>231</width>
       <height>31</height>
      </rect>
     </property>
     <property name="text">
      <string>Guardar Consulta</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_7">
     <property name="geometry">
      <rect>
       <x>90</x>
       <y>440</y>
       <width>231</width>
       <height>21</height>
      </rect>
     </property>
     <property name="text">
      <string>Pruebas realizadas</string>
     </property>
    </widget>
    <widget class="QTextEdit" name="inputPruebasRealizadas">
     <property name="geometry">
      <rect>
       <x>90</x>
       <y>470</y>
       <width>221</width>
       <height>171</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">background-color: rgb(255, 255, 255);</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_8">
     <property name="geometry">
      <rect>
       <x>390</x>
       <y>430</y>
       <width>151</width>
       <height>31</height>
      </rect>
     </property>
     <property name="text">
      <string>Diagnóstico</string>
     </property>
    </widget>
    <widget class="QTextEdit" name="inputDiagnostico">
     <property name="geometry">
      <rect>
       <x>390</x>
       <y>470</y>
       <width>231</width>
       <height>171</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">background-color: rgb(255, 255, 255);</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_9">
     <property name="geometry">
      <rect>
       <x>700</x>
       <y>440</y>
       <width>151</width>
       <height>21</height>
      </rect>
     </property>
     <property name="text">
      <string>Tratamiento</string>
     </property>
    </widget>
    <widget class="QTextEdit" name="inputTratamiento">
     <property name="geometry">
      <rect>
       <x>700</x>
       <y>470</y>
       <width>231</width>
       <height>171</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">background-color: rgb(255, 255, 255);</string>
     </property>
    </widget>
    <widget class="QComboBox" name="inputMotivoConsulta">
     <property name="geometry">
      <rect>
       <x>750</x>
       <y>110</y>
       <width>271</width>
       <height>31</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">background-color: rgb(255, 255, 255);
color: rgb(0, 0, 0);</string>
     </property>
     <item>
      <property name="text">
       <string/>
      </property>
     </item>
     <item>
      <property name="text">
       <string>Consulta General</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>Examenes</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>Curaciones</string>
      </property>
     </item>
    </widget>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1150</width>
     <height>21</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuMenu">
    <property name="title">
     <string>Menú</string>
    </property>
    <addaction name="separator"/>
    <addaction name="actionRegresar"/>
   </widget>
   <addaction name="menuMenu"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <action name="actionRegresar">
   <property name="text">
    <string>Regresar</string>
   </property>
  </action>
 </widget>
 <resources>
  <include location="../Imagenes/Imagen.qrc"/>
 </resources>
 <connections/>
</ui>
